ui/app/src/translations/locales/es.json

```diff
@@ -525,7 +525,7 @@
   "itemState.scheduledForStaging": "Programado para puesta en escena",
   "itemState.scheduledToGoLive": "Programado para vivir",
   "itemState.submitted": "Presentado",
-  "itemState.submittedToLive": "Enviado para vivir",
+  "itemState.submittedToLive": "Enviado a vivir",
```

Comment:
Update the translation to "Enviado a producción" or some better translation for live